content/40.inequality.md

```diff
@@ -1,67 +1,125 @@
-It should be noted that comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally [@doi:10.1073/pnas.2008760117].
-Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities [@doi:10.1111/jgs.16472; @url:https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019-(covid-19)]. <!-- will this url work for Manubot?-->
+Dementia and cancer were also associated with the risk of death in an analysis of a large number (more than 20,000) of COVID-19 patients in the United Kingdom [@doi:10.1136/bmj.m1985].
+
+These health conditions affect biology in ways that are thought to overlap with the pathogenesis of COVID-19.
+Analyses investigating possible functional relationships between diabetes and COVID-19 have revealed that _ACE2_ expression may be elevated in individuals with diabetes mellitus [@doi:10.2337/dc20-0643] and that ketoacidosis, a metabolic disorder typically associated with diabetes, is observed in COVID-19 patients with and without diabetes [@doi:10.1111/dom.14057].
+These findings suggest that the pathology of diabetes may overlap with the pathology of COVID-19 [@doi:10.1152/ajpendo.00124.2020].
+Although a clear functional connection between obesity and COVID-19 outcomes is not yet established [@doi:10.1016/j.medj.2020.06.005], obesity is correlated with other comorbid conditions such as hypertension, diabetes, and pulmonary disease, as well as to chronic inflammation [@doi:10.1007/s11695-020-04677-z].
+Structural racism has been proposed as a driver of both obesity and severe impacts of COVID-19 [@doi:10.1016/j.cmet.2021.01.010].
+Dementia and other forms of cognitive decline are thought to be functionally related to diabetes [@doi:10/c3tm55], and both cancer and dementia are related to chronic inflammation [@doi:10.1038/nature01322; @doi:10.1186/s13195-015-0117-2].
+
+It should be noted that such comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally [@doi:10.1073/pnas.2008760117].
+Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities [@doi:10.1111/jgs.16472; @url:https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019-(covid-19)]. <!--To Do: will this url work for Manubot?-->
```

Comment:
No, that URL won't work.  I can add an alias (tag) on your branch as a workaround.

Reply:
Hopefully 11f1d8a and 265daea will fix this.  I'll check the build artifact.